onboarding.go

```diff
@@ -28,12 +28,13 @@ type OnboardingResults struct {
 // OnboardingRequest is the request
 // to setup defaults.
 type OnboardingRequest struct {
-	User            string        `json:"username"`
-	Password        string        `json:"password"`
-	Org             string        `json:"org"`
-	Bucket          string        `json:"bucket"`
-	RetentionPeriod time.Duration `json:"retentionPeriodHrs,omitempty"`
-	Token           string        `json:"token,omitempty"`
+	User                   string        `json:"username"`
+	Password               string        `json:"password"`
+	Org                    string        `json:"org"`
+	Bucket                 string        `json:"bucket"`
+	RetentionPeriodSeconds int64         `json:"retentionPeriodSeconds,omitempty"`
+	RetentionPeriodHours   time.Duration `json:"retentionPeriodHrs,omitempty"`
```

Comment:
Self-review: I meant to name this `RetentionPeriodDeprecated` to emphasize that it's not actually hours